crates/nu-std/lib/help.nu

```diff
@@ -98,11 +98,17 @@ def "nu-complete list-externs" [] {
     $nu.scope.commands | where is_extern | select name usage | rename value description
 }
 
+def help-header [
```

Comment:
Could you use `colorize` instead of adding another command?

Reply:
agree, or `help-header` could use `colorize`.

Reply:
Will use `colorize` kind of everywhere I can by the time the PR is finalized...